.github/workflows/build_release.yml

```
@@ -0,0 +1,175 @@
+name: "Package and Release"
+
+on:
```

yamllint

[warning] truthy value should be one of [false, true]

(truthy)